### DISCUSSION TITLE
Sharing specific files or folder access?

### DISCUSSION THREAD

**Discussion by ryanwinchester** [spacedriveapp/spacedrive#1623] [General]
I have been generating a backlog of video content and looking for a way to share them with an editor (the person editing my videos) and this seems like it might be able to offer a solution.

Will I be able to share specific Video Files/Folders with someone else without having to host them in the cloud?

Would you be able to share folder access with other people (or spacedrive users)?

**Comment by utkubakir** [comment]
Hi, I think what you are explaining fits what we call "Spaces". You will be able to host any folder or file with it with anyone you like. In the future, you can tag the video file you want and it will automatically copy itself to this space and can be viewed/downloaded by your editor. Your space will be online whenever you are (or you could host your space with us so it's online 24/7)
